cloudevents/spec.md

```diff
@@ -178,6 +178,12 @@ CloudEvents attribute names MUST consist of lower-case letters ('a' to 'z') or
 digits ('0' to '9') from the ASCII character set. Attribute names SHOULD be
 descriptive and terse and SHOULD NOT exceed 20 characters in length.
 
+#### Reserved Names
+
+The following attribute names are reserved and MAY NOT be used:
```

Comment:
Should "MAY NOT" be "MUST NOT"? (RFC 2119 doesn't include MAY NOT.)

Possibly "The following names are reserved and MUST NOT be used for attributes"? (They're not attribute names, arguably...)

Reply:
ack